src/hotspot/share/gc/shenandoah/shenandoahHeap.cpp

```diff
@@ -276,7 +276,7 @@ jint ShenandoahHeap::initialize() {
       os::commit_memory_or_exit(verify_bitmap.base(), verify_bitmap.size(), bitmap_page_size, false,
                                 "Cannot commit verification bitmap memory");
     }
-    MemTracker::record_virtual_memory_type(verify_bitmap.base(), mtGC);
+    MemTracker::record_virtual_memory_type(verify_bitmap.base(), verify_bitmap..size(), mtGC);
```

Comment:
Does this even compile?

Reply:
No, you're right. Shenandoah is not the main build.
I thought that GHA builds does this but seems not.
Fixed. Thanks.